data/spells/scripts/house/edit_door_list.lua

```diff
@@ -0,0 +1,33 @@
+local function getNextPosition(player)
```

Comment:
I think you should remove this redundant function, and use Position.getNextPosition instead.